website/docs/learn/5-deploy-your-project.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Deploy your project
+id: deploy-your-project
+description: In this part of the tutorial, we'll go through how you can deploy your project with dbt Cloud.
+---
+
+import Alert from '@site/src/components/alert';
+import Lightbox from '@site/src/components/lightbox';
+import LoomVideo from '@site/src/components/loom';
+import Collapsible from '@site/src/components/collapsible';
+import CloudCore from '@site/src/components/cloudcore';
+import FAQ from '@site/src/components/faqs';
+
+As the `jaffle_shop` business gains more customers, and those customers create more orders, there will be more records added to your source data. Since the `customers` model is materialized as a table, you'll need to periodically rebuild your table to ensure that the data stays up-to-date.
+
+This is often referred to as "deploying a project" or "[running a project in production](https://docs.getdbt.com/docs/running-dbt-in-production)". In this part of the tutorial, we'll go through how you can deploy your project with dbt Cloud.
+
+## Commit your changes
+First off, we need to commit the changes we made to our project so that our repository has our latest code.
+
+### dbt Cloud
+<LoomVideo id="afd55d89abdc4a77b34deaee90da0813" />
+
+Click the `commit` button, with a message like "Add customers model"
+
+### dbt Core
```

Comment:
```suggestion
### dbt CLI
```
? I noticed this in the main tutorial as well

Reply:
Thanks!